### DISCUSSION TITLE
LAB 05.01 Task 2: Simulated prediction signal

### DISCUSSION THREAD

**Discussion by soyudea** [rramosp/ai4eng.v1#249] [03 - GENERACION DE MODELOS]
Profesor en esta tarea dan un arreglo  con valores signal, y dan un valor accuraccy aleatorio, encontrar un vector de predicciones p, tal que si la predicción es perfecta se encontrara un valor PNL con todos sus valores positivos.

p se encuentra randonicamente y entiendo que la cantidad de aciertos debe ser igual al acuracy, pero luego debo permutar sus valores hasta obtener la predicción perfecta.

Hacer permutaciones de un arreglo de 20 valores es inmenso, y la verdad he fracasado al intentar un algoritmo que encuentre el p pedido.

mi whatsApp es el 3043440112

Por donde me puedo meter

gracias

**Comment by imEag** [comment]
Solucionado en clase
